examples/pytorch_platform_handler/client.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+# Copyright 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import os
+import sys
+
+import numpy as np
+from PIL import Image
+from tritonclient import http as httpclient
+from tritonclient.utils import *
+
+script_directory = os.path.dirname(os.path.realpath(__file__))
+
+server_url = "localhost:8000"
+model_name = "resnet50_pytorch"
+input_name = "INPUT"
+output_name = "OUTPUT"
+label_path = os.path.join(script_directory, "resnet50_labels.txt")
+image_path = os.path.join(script_directory, "mug.jpg")
```

Comment:
I think we need to clarify where `mug.jpg` is coming from, since it is not among the files of this example

Reply:
Added link.
The link is included in the download part of the docs. If the user follows the docs step-by-step, the image will be present when the client is ran.

Reply:
I meant, we can mention that the `mug.jpg` image comes inside the container, if I remember correctly.

Reply:
[List mug.jpg paths](https://github.com/triton-inference-server/python_backend/pull/287/commits/a9cc34d8756b3ec8fdbd07470ba5163460e6fb55)

Just to clarify, from the quick start guide perspective, the user is not expected look inside the container for the image. If the user follows the guide step-by-step, the image will be placed on the `script_directory` on the `client.py`.

Reply:
I think we can specify in comments something like:
" Note: Provided client script assumes that the user is following steps, described in a corresponding README. Thus, make sure that `mug.jpg` is in the same location as `client.py`. This image is available at <link>. Alternatively,
`mug.jpg` also comes with our sdk container at the following location: /workspace/images/mug.jpg. If using the provided image, make sure to adjust `image_path` to the correct value." 

Feel free to adjust accordingly.

Reply:
Sure. I kept it short:
```
# The 'mug.jpg' image will be present at the script_directory if the steps on
# the provided README.md are followed. The image may also be found at
# '/workspace/images/mug.jpg' on the SDK container or
# '/opt/tritonserver/qa/images/mug.jpg' on the QA container.
```